### PR TITLE
IBX-8138: [Rector] Applied rules from Symfony 5 Rector set lists

### PR DESCRIPTION
**Pull request by alongosz**
| :ticket: Issue | IBX-8138 |
|----------------|-----------|

#### Description:

Applied rules from Symfony 5 Rector set lists.

#### For QA:

No QA needed. Regression tests should be enough.

#### Documentation:

This is code refactoring without behavior change. No documentation changes needed.

### CODE REVIEW

File: src/lib/Core/Debug/Command/GoBackCommand.php [hunk 1]
```diff
@@ -33,12 +33,12 @@ protected function configure()
...
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $this->session->back();
 
         $output->writeln("The last page from browser's history has been visited.");
 
-        return 0;
+        return \Symfony\Component\Console\Command\Command::SUCCESS;
```
Comment (konradoboza) [inline src/lib/Core/Debug/Command/GoBackCommand.php:42]: Import.

File: src/lib/Core/Debug/Command/RefreshPageCommand.php [hunk 2]
```diff
@@ -33,12 +33,12 @@ protected function configure()
...
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $this->session->reload();
 
         $output->writeln('Page has been refreshed.');
 
-        return 0;
+        return \Symfony\Component\Console\Command\Command::SUCCESS;
```
Comment (konradoboza) [inline src/lib/Core/Debug/Command/RefreshPageCommand.php:42]: Import.

File: src/lib/Core/Debug/Command/ShowHTMLCommand.php [hunk 3]
```diff
@@ -33,10 +33,10 @@ protected function configure()
             ->setHelp('');
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $output->writeln($this->session->getPage()->getOuterHtml());
 
-        return 0;
+        return \Symfony\Component\Console\Command\Command::SUCCESS;
```
Comment (konradoboza) [inline src/lib/Core/Debug/Command/ShowHTMLCommand.php:40]: Import.

File: src/lib/Core/Debug/Command/ShowURLCommand.php [hunk 4]
```diff
@@ -33,10 +33,10 @@ protected function configure()
             ->setHelp('');
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $output->writeln($this->session->getCurrentUrl());
 
-        return 0;
+        return \Symfony\Component\Console\Command\Command::SUCCESS;
```
Comment (konradoboza) [inline src/lib/Core/Debug/Command/ShowURLCommand.php:40]: Import.

File: src/lib/Core/Debug/Command/TakeScreenshotCommand.php [hunk 5]
```diff
@@ -54,7 +54,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $response = Uploader::unsigned_upload($filePath, self::PRESET);
             $output->writeln(sprintf('Open image at %s', $response['secure_url']));
 
-            return 0;
+            return \Symfony\Component\Console\Command\Command::SUCCESS;
```
Comment (konradoboza) [inline src/lib/Core/Debug/Command/TakeScreenshotCommand.php:57]: Import.

File: src/lib/Core/Debug/Command/RefreshPageCommand.php (hunk 2 above)
Comment (Steveb-p) [reply src/lib/Core/Debug/Command/RefreshPageCommand.php:42]: or 
```suggestion
        return self::SUCCESS;
```

**Review by Steveb-p — COMMENTED** [review]
Valid PHPStan error.

**Comment by micszo** [comment]
@alongosz @konradoboza @Steveb-p thank you for opening and reviewing. Appreciated! 🖤

**Review by konradoboza — APPROVED** [review]

**Comment by tomaszszopinski** [comment]
Behat should be green now, we can proceed with merge.
greenbutatwhatcost

**Review by tomaszszopinski — APPROVED** [review]
QA approved.

**Comment by alongosz** [comment]
Ok, it seems to be safe to merge this one now, as the changes don't affect extension points.

**Pull request merged**